teerex/src/tests/tests.rs

```diff
@@ -676,30 +661,21 @@ fn update_block_confirmation_works() {
 		));
 		assert_eq!(Teerex::enclave_count(), 1);
 
-		assert_ok!(Teerex::confirm_block(
+		assert_ok!(Teerex::confirm_proposed_sidechainblock(
 			Origin::signed(signer7.clone()),
 			shard7.clone(),
 			block_hash.clone(),
-			ipfs_hash.as_bytes().to_vec()
-		));
-
-		let expected_event = Event::Teerex(RawEvent::UpdatedIpfsHash(
-			shard7.clone(),
-			1,
-			ipfs_hash.as_bytes().to_vec(),
 		));
-		assert!(System::events().iter().any(|a| a.event == expected_event));
 
-		let expected_event = Event::Teerex(RawEvent::BlockConfirmed(signer7.clone(), block_hash));
+		let expected_event = Event::Teerex(RawEvent::ProposedSidechainBlock(signer7, block_hash));
 		assert!(System::events().iter().any(|a| a.event == expected_event));
 	})
 }
 
 #[test]
-fn verify_block_confirmation_from_shards_not_enclave_fails() {
+fn verify_confirm_proposed_sidechainblock_from_shard_neq_mrenclave_fails() {
```

Comment:
Instead of `verify ... fails`, I would suggest: `confirm.....mrenlcave_errs()`. I have seen this pattern also in other codebases.

Reply:
Done 👍